Client/Coordinators/Scene/SceneCoordinator.swift

```diff
@@ -9,10 +9,11 @@ import Shared
 /// Each scene has it's own scene coordinator, which is the root coordinator for a scene.
 class SceneCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, LaunchFinishedLoadingDelegate {
     var window: UIWindow?
+    private let screenshotService = ScreenshotService()
 
```

Comment:
How do we decide when we instantiate like this or if we pass it from a dependency container?